.circleci/config.yml

```diff
@@ -26,9 +26,20 @@ jobs:
               git checkout -qf FETCH_HEAD
             fi
       - install_container_dependencies
+      - restore_cache:
+          keys:
```

Comment:
This could just be `key`, since we don't have multiple keys.

Reply:
Done.